api/Navigator.json

```diff
@@ -1572,7 +1572,7 @@
           },
           "status": {
             "experimental": false,
-            "standard_track": false,
+            "standard_track": true,
```

Comment:
Confirmed: https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-oscpu

(But required to only be supported in Gecko.)